python/src/ClaytonCopulaFactory_doc.i.in

```diff
@@ -1,6 +1,12 @@
 %feature("docstring") OT::ClaytonCopulaFactory
 "Clayton Copula factory.
 
+
+Available constructor:
+    ClaytonCopulaFactory()
```

Comment:
I think we dont have a policy when there is no argument:
https://stackoverflow.com/questions/60793794/python-google-style-docstring-for-function-with-no-arguments
but I would drop the available ctor list as well, like in the case where there is only one ctor with arguments